extensions/wikia/PortabilityDashboard/PortabilityDashboard.i18n.php

```diff
@@ -29,7 +29,8 @@
 	'portability-dashboard-language-filter-all' => 'all',
 	'portability-dashboard-special-insights-template-without-title' => 'Unorganized templates on Special:Insights',
 	'portability-dashboard-special-insights-custom-infobox-title' => 'Non-portable infoboxes on Special:Insights',
-	'portability-dashboard-refresh-frequency-info' => 'Portability data, content page views and migration impact are recalculated every Saturday at 9:00 GMT.'
+	'portability-dashboard-refresh-frequency-info' => 'Portability data, content page views and migration impact are recalculated every Saturday at 9:00 GMT.',
+	'portability-dashboard-no-results-info' => 'No results have been found for selected language.'
```

Comment:
I recommend to update to match the description in the tickets more:
"We don't have any wikis in selected language that are in top {param} most significant communities from the migration impact perspective."